extractor/src/extractors/JsonnetJsonExtractor.ts

```diff
@@ -0,0 +1,61 @@
+import {Dependency, DependencyManagementFile} from "@depscloud/api/v1alpha/deps";
+import MatchConfig from "../matcher/MatchConfig";
+import Extractor from "./Extractor";
+import ExtractorFile from "./ExtractorFile";
+import Languages from "./Languages";
+
+const extract = (dependency: any,scopes:any):Dependency => {
+    return {
+        organization:"",
+        module:"",
+        versionConstraint: dependency.version,
+        scopes:[scopes],
+        name: parseName(dependency),
+    };
+}
+
+//The name should be formatted as {{ remote }}[#{{ subdir }}] where subdir
+const parseName = (dependency) => {
+    return `${dependency.source.git.remote}[#${dependency.source.git.subdir}]`;
+}
+
+    
+
+class JsonnetJsonExtractor implements Extractor {
+    matchConfig(): MatchConfig {
+        return {
+            includes: [
+                "**/jsonnetfile.json",
+            ],
+            excludes: [],
+        };
+    }
+    requires(): string[] {
+        return [ "jsonnetfile.json" ];
+    }
+    public async extract(url: string, files: { [key: string]: ExtractorFile }): Promise<DependencyManagementFile> {
+        const {
+            name,
+            version,
+            organization,
+            dependencies,
+
+        } = files["jsonnetfile.json"].json();
+
+        const deps = dependencies.map(dependence => extract(dependence , "dependencies"));
+
+
+        return {
+            language: Languages.JAVASCRIPT,
+            system:"",
+            sourceUrl: "",
+            organization,
+            module: "",
+            version,
+            dependencies: deps,
+            name,
```

Comment:
`name` should be `url`

organization / module / sourceURL shouldn't be set.

Reply:
`name should be url` what do you mean by url ?

Reply:
So jsonnet-bundler references libraries using their vcs url and an optional subdir. The way I interpret this into our standardized model is that the module's `name` is the URL of the repo and the `scope` is the `subdir`